data/mods/Magiclysm/npc/TALK_FORGE_HELEN_TAVREL.json

```diff
@@ -0,0 +1,112 @@
+[
+  {
+    "type": "npc",
+    "id": "forge_lord_reagents",
+    "//": "The Lord of the Forge of Wonders.",
+    "name_unique": "Helen Tavrel, Former Pirate",
+    "name_suffix": "Forge Lord",
+    "class": "NC_FORGE_LORD_REAGENT",
+    "attitude": 0,
+    "mission": 3,
+    "gender": "female",
+    "chat": "TALK_FORGE_LORD_REAGENT",
+    "faction": "forge_lords"
+  },
+  {
+    "type": "npc_class",
+    "id": "NC_FORGE_LORD_REAGENT",
+    "name": { "str": "Pirate Queen" },
```

Comment:
i almost feel like the forge lords could use some spells, but not necessary in this pr specifically.

Reply:
Spells for the PC to learn? I can put that on my to do list.

Reply:
both really, since #49800 is in the near-ish future.